data/json/professions.json

```
@@ -4364,6 +4415,277 @@
       ]
     }
   },
+  {
+    "type": "profession",
+    "id": "mil_marksman",
+    "name": "Military Marksman",
+    "description": "You like to think you're a real sniper, but really you're just infantry with a bigger gun.",
+    "points": 4,
+    "skills": [ { "level": 1, "name": "survival" }, { "level": 2, "name": "gun" }, { "level": 2, "name": "rifle" } ],
+    "items": {
+      "both": {
+        "items": [
+          "hat_boonie",
+          "jacket_army",
+          "pants_army",
+          "longshirt",
+          "rucksack",
+          "knee_pads",
+          "elbow_pads",
+          "socks",
+          "boots_combat",
+          "gloves_tactical",
+          "ballistic_vest_esapi"
+        ],
+        "entries": [
+          { "item": "m14ebr", "ammo-item": "762_51", "charges": 20, "contents-item": [ "acog_scope", "shoulder_strap", "bipod" ] },
+          { "item": "ear_plugs", "custom-flags": [ "no_auto_equip" ] },
+          { "item": "tacvest", "contents-group": "army_mags_m14" },
+          { "item": "knife_combat", "container-item": "sheath" },
+          { "item": "m17", "ammo-item": "9mm", "charges": 17, "container-item": "holster" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "boy_shorts", "sports_bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "mil_auto_rifleman",
+    "name": "Military Automatic Rifleman",
+    "description": "S stands for suppressing fire!",
+    "points": 3,
```

```suggestion
    "points": 5,
```
200 hundred rounds of 5.56 is a lot to start with so I've increased the points